psqlgraph/query.py

```diff
@@ -438,16 +441,14 @@ def null_props(self, keys=[], *args):
             g.null_props(['key1', 'key2', 'key3']).count()
 
         """
-
-        keys = keys if hasattr(keys, '__iter__') else [keys]
```

Comment:
the new behavior isn't really covering a case when the key is not of type `str`... so you might want to change the logic around converting `foo` to `[foo]` based on `isinstance(foo, str)` to something like: `foo = foo if iterable(foo) and not isinstance(foo, str) else [str(foo)]`

Reply:
key can either be a string or a list of strings, so one either check it its a list or check if its a string. Checking for string seemed cleaner to me